### ISSUE TITLE
Add a Security Policy to the Uli repository

### ISSUE THREAD

**Issue by dennyabrain** [tattle-made/Uli#293]
**Is your feature request related to a problem? Please describe.**
I was going through the community standards checklist provided by Uli and noticed that we are missing a security policy. 

**Describe the solution you'd like**
Here's github's [guide](
https://docs.github.com/en/code-security/getting-started/adding-a-security-policy-to-your-repository) on how to do it. Review and find out what uli specific things we could add besides the boilerplate.

**Additional context**
See if the work done on the content security policy or sonar qube is relevant to this.

**Comment by duggalsu** [comment]
There are different types of company-wide security policies that can be implemented
See - https://www.sans.org/information-security-policy/

For disclosing security issues, we are specifically looking at a 'Vulnerability Disclosure Policy'.
- https://cheatsheetseries.owasp.org/cheatsheets/Vulnerability_Disclosure_Cheat_Sheet.html
```
Researchers should:
- Ensure that any testing is legal and authorised.
- Respect the privacy of others.
- Make reasonable efforts to contact the security team of the organisation.
- Provide sufficient details to allow the vulnerabilities to be verified and reproduced.
- Not demand payment or rewards for reporting vulnerabilities outside of an established bug bounty program.

Organisations should:
- Provide a clear method for researchers to securely report vulnerabilities.
- Clearly establish the scope and terms of any bug bounty programs.
- Respond to reports in a reasonable timeline.
- Communicate openly with researchers.
- Not threaten legal action against researchers.
- Request CVEs where appropriate.
- Publish clear security advisories and changelogs.
- Offer rewards and credit.
```

Examples of disclosure policies
- https://about.google/appsecurity/
- https://www.hackerone.com/disclosure-guidelines
- https://about.gitlab.com/security/disclosure/
- https://www.okta.com/vulnerability-reporting-policy/
- https://razorpay.com/responsible-disclosure/
- https://www.nykaa.com/responsible-disclosure

Other References:
- https://googleprojectzero.blogspot.com/2020/01/policy-and-disclosure-2020-edition.html
- https://www.hackerone.com/vulnerability-management/vulnerability-disclosure-policy-basics-5-critical-components
- https://owasp.org/www-community/vulnerabilities/Vulnerability_template
- https://securityboulevard.com/2021/01/how-to-write-a-vulnerability-management-policy/

**Comment by duggalsu** [comment]
Merged with #405
